packages-native/animation/src/ui/Styles.ts

```diff
@@ -0,0 +1,12 @@
+import { Style } from "@native-mobile-resources/util-widgets";
+import { ViewStyle } from "react-native";
+
+export interface AnimationStyle extends Style {
+    container: ViewStyle;
+}
+
+export const defaultAnimationStyle: AnimationStyle = {
+    container: {
+        // backgroundColor: "green"
```

Comment:
maybe remove the commented out code ? If you want to give an example:
```
/* Example:
/* backgroundColor: "green"
*/
```

Reply:
oopsy, thanks